checker/jdk/index/src/java/io/File.java

```diff
@@ -195,7 +195,7 @@ int getPrefixLength() {
      * string for convenience.  This string contains a single character, namely
      * <code>{@link #separatorChar}</code>.
      */
-    public static final @MinLen(1) String separator = "" + separatorChar;
+    public static final /*!MinLen(1)*/ String separator = "" + separatorChar;
```

Comment:
I left it in a comment because it would be correct if the Index Checker could handle Strings. But since MinLen is now a Value Checker annotation, maybe it should just be deleted.